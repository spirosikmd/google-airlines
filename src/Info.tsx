import React from 'react';
import Anchor from './Anchor';

const Info = () => (
  <>
    <h1 className="text-4xl font-medium">Google Airlines</h1>
    <p className="max-w-2xl text-gray-600">
      Google created their own airline logos for{' '}
      <Anchor href="https://www.google.com/flights">google flights</Anchor> for
      many airlines out there. This is a collection of a few of them in a random
      order. I got the IATA codes from{' '}
      <Anchor href="https://en.wikipedia.org/wiki/List_of_airline_codes">
        wikipedia
      </Anchor>
      . Enjoy!
    </p>
  </>
);

export default Info;
