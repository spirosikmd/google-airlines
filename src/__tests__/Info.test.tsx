import React from 'react';
import { render, cleanup } from 'react-testing-library';
import Info from '../Info';

afterEach(cleanup);

test('renders info', () => {
  const { asFragment } = render(<Info />);

  expect(asFragment()).toMatchSnapshot();
});
