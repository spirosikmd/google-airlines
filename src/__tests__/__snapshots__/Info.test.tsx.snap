// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders info 1`] = `
<DocumentFragment>
  <h1
    class="app__title"
  >
    Google Airlines
  </h1>
  <p
    class="app__info"
  >
    Google created their own airline logos for 
    <a
      class="app__anchor"
      href="https://www.google.com/flights"
      target="_blank"
    >
      google flights
    </a>
     for many airlines out there. This is a collection of a few of them in a random order. I got the IATA codes from 
    <a
      class="app__anchor"
      href="https://en.wikipedia.org/wiki/List_of_airline_codes"
      target="_blank"
    >
      wikipedia
    </a>
    . Enjoy!
  </p>
</DocumentFragment>
`;
